f# web sdk fails publish after updated web.sdk

After #4948 was merged the `dotnet publish` of a f# web fails.

## Steps to reproduce

preview4 branch 

- `dotnet new -l fsharp -t web`
- `dotnet restore`
- `dotnet publish`

## Expected  behavior

Ok

## Actual behavior

Error:

```
C:\dotnetcli\dotnet-dev-win-x64.1.0.0-preview4-004233\sdk\1.0.0-preview4-004233\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(100,5): error MSB3094: "DestinationFiles" refers to 1 item(s), and "SourceFiles" refers to 2 item(s). They must have the same number of items. [e:\temp\web4f\w\w.fsproj]
```

## Environment data
`dotnet --info` output:

```
.NET Command Line Tools (1.0.0-preview4-004233)

Product Information:
 Version:            1.0.0-preview4-004233
 Commit SHA-1 hash:  8cec61c6f7

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\dotnetcli\dotnet-dev-win-x64.1.0.0-preview4-004233\sdk\1.0.0-preview4-004233
```

--- a/src/dotnet/commands/dotnet-new/FSharp_Web/$projectName$.fsproj
+++ b/src/dotnet/commands/dotnet-new/FSharp_Web/$projectName$.fsproj
@@ -10,20 +10,10 @@
     <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wp8+wpa81;</PackageTargetFallback>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <GlobalExclude>$(GlobalExclude);bin\**;obj\**;node_modules\**;jspm_packages\**;bower_components\**;**\*.user;**\*.*proj</GlobalExclude>
-  </PropertyGroup>
-
   <ItemGroup>
-    <None Include="**\*" />
     <Compile Include="Controllers\*.fs" />
     <Compile Include="Startup.fs" />
     <Compile Include="Program.fs" />
-    <EmbeddedResource Include="**\*.resx" />
-    <Content Include="wwwroot\**" CopyToPublishDirectory="PreserveNewest" />
-    <Content Include="**\*.cshtml" Exclude="wwwroot\**\*.cshtml" CopyToPublishDirectory="PreserveNewest" />
-    <Content Include="**\*.config" Exclude="wwwroot\**\*.config" CopyToPublishDirectory="PreserveNewest" />
-    <Content Include="**\*.json" Exclude="wwwroot\**\*.json" CopyToPublishDirectory="PreserveNewest" />
   </ItemGroup>
 
   <ItemGroup>

